server/server.js

```diff
@@ -38,6 +38,16 @@ function prodLangs() {
 
 const availableLanguages = conf.l10n_dev ? allLangs() : prodLangs();
 
+const envURL = (env) => {
+  switch (env) {
+    case 'test':
+      return 'https://send.stage.mozaws.net';
```

Comment:
I'm not convinced this is a thing.
I have a feeling that stage would use an env of "production". Not sure what [if anything] uses the "test" NODE_ENV, unless Circle-CI somehow sets it to that, but I strangely kind of doubt it.

I don't know how to tell the detect whether we're running on stage vs prod on the server.